sentry.client.config.js

```diff
@@ -2,25 +2,10 @@
 // The config you add here will be used whenever a page is visited.
 // https://docs.sentry.io/platforms/javascript/guides/nextjs/
 
-import { Replay } from '@sentry/browser';
 import * as Sentry from '@sentry/nextjs';
 
 import defaultConfig from './sentry.default.config.js';
 
 Sentry.init({
   ...defaultConfig,
-  // This sets the sample rate to be 10%. You may want this to be 100% while
-  // in development and sample at a lower rate in production
-  replaysSessionSampleRate: 0,
-  // If the entire session is not sampled, use the below sample rate to sample
-  // sessions when an error occurs.
-  replaysOnErrorSampleRate: 0.01,
-
-  integrations: [
-    new Replay({
-      // Additional SDK configuration goes in here, for example:
-      maskAllText: true,
-      blockAllMedia: true,
-    }),
-  ],
```

Comment:
What are we doing here?

Reply:
Since we are not using [sentry replay](https://sentry.io/for/session-replay/) I removed the integration, this reduces the bundle imported by sentry.